### DISCUSSION TITLE
some users can't open hubs on their computer when this is a custom client ?

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#2785] [General Questions & Discussion] [answered]
Is there some port that the network administrator can close , Because I have deliver a custom hubs to a compagny and the users can 't open the hubs . I have done a custom client in AWS, and I can' t resolve this problem of connexion . 
Thx for your reply  

Marc

**Comment by brianpeiris** [comment, marked as answer]
Hi @bemyevent. Hubs uses WebRTC for voice communication as well as WebSockets for networking other data. It's typically WebRTC that is blocked in restrictive networks like offices or educational institutions. You might want to try checking your firewall as described here: https://hubs.mozilla.com/docs/hubs-troubleshooting.html#unable-to-connect-error
It might also help to test WebRTC with this generic troubleshooter: https://test.webrtc.org/
